scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala

```diff
@@ -215,13 +217,21 @@ package object sparkey extends SparkeyReaderInstances {
     def asSparkey(
       path: String = null,
       maxMemoryUsage: Long = -1,
-      numShards: Short = DefaultNumShards
+      numShards: Short = DefaultNumShards,
+      compressionType: CompressionType = CompressionType.NONE,
```

Comment:
🎨 Since these defaults are used in multiple methods might be worth moving them to a constant.